tools/src/commands/Vendor.ts

```diff
@@ -18,23 +25,42 @@ import {
   TransformFilesContent,
   TransformFilesName,
 } from '../vendoring/devmenu';
+import { GenerateJsonFromPodspec } from '../vendoring/devmenu/steps/GenerateJsonFromPodspec';
+import { MessageType, Print } from '../vendoring/devmenu/steps/Print';
 import { RemoveFiles } from '../vendoring/devmenu/steps/RemoveFiles';
+import { toRepoPath } from '../vendoring/devmenu/utils';
+
+async function getRequierdIOSVersion(): Promise<string> {
```

Comment:
We usually write `Ios` in such cases. It's also the rule in EAS code, so I think it would be nice to stick to this rule here as well.

```suggestion
async function getRequiredIosVersion(): Promise<string> {
```